timelock/src/test.rs

```diff
@@ -38,34 +37,11 @@ fn create_claimable_balance_contract(e: &Env) -> ClaimableBalanceContractClient
     ClaimableBalanceContractClient::new(e, contract_id)
 }
 
-fn sign_args(
```

Comment:
I think we should either cover non-source-account signatures or not support them. If we want to keep examples simple we should probably go with the latter (then we can just remove `Signature` argument from the contract )

Reply:
The contract should support all types of auth. The tests just show the source account model since that's what we expect people to use for meridian. We can add the non-source uses of auth later once we finalize what complex auth should look like.